src/pages/journal/journal-components/journal-components.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { IonicModule } from 'ionic-angular';
+
+import { JournalTestDetailsComponent } from './journal-test-details/journal-test-details';
+import { JournalTimeComponent } from './journal-time/journal-time';
+import { JournalCandidateComponent } from './journal-candidate/journal-candidate';
+import { JournalTestOutcomeComponent } from './journal-test-outcome/journal-test-outcome';
+import { JournalSlotComponent } from './journal-slot/journal-slot';
+
+@NgModule({
+	declarations: [JournalTestDetailsComponent, JournalTimeComponent, JournalCandidateComponent, JournalTestOutcomeComponent, JournalSlotComponent],
```

Comment:
Could you split out some lines on `declarations` and `exports` here?

Reply:
Done